Fastfile

```diff
@@ -84,7 +84,7 @@ platform :ios do
     sync_code_signing(type: "adhoc", app_identifier: [app_identifier])
     bump
     build_app(scheme: scheme, configuration: configuration, export_method: "ad-hoc")
-    firebase_app_distribution(app: firebase_app, groups: firebase_group)
+    firebase_app_distribution(app: firebase_app, groups: firebase_group, release_notes: options[:changelog])
```

Comment:
@pedrohperalta is the idea to get the plain text changelog message from the command line execution or reading it from the `CHANGELOG.md` file?

Reply:
@barbosa the idea is to read it from the `CHANGELOG.md`. However, the function that calls `firebase_beta` should decide it, the string should only be a dependency of this lane.